Library/Formula/postgresql.rb

```diff
@@ -77,8 +82,16 @@ def install
   end
 
   def post_install
-    unless File.exist? "#{var}/postgres"
-      system "#{bin}/initdb", "#{var}/postgres"
+    system "pkill postgres || true"
+
+    if (var/"postgres").exist?
+      mv var/"postgres", var/"postgres-#{version_suffix}"
```

Comment:
As in for the ` postinstall `? Won't that break when ` 9.5 ` comes out and ` PG_VERSION ` is no longer correct in that file? Apologies if I'm being incredibly dense here, didn't sleep overly well and the brain hasn't quite fired up yet :sleeping:.